static/js/tools/timeline_page.test.tsx

```diff
@@ -63,9 +64,59 @@ test("Single place and single stats var", () => {
           `<button class="mdl-chip__action"><i class="material-icons">` +
           `cancel</i></button></div></div></div>`
       );
+
+      // add one statsVar by clicking the checkbox
+      wrapper.find("#drill .checkbox").at(0).simulate("click");
+      Promise.resolve(wrapper)
+        .then(() => wrapper.update())
+        .then(() => {
+          wrapper.update();
+          expect(window.location.hash).toEqual(
+            "place=geoId%2F05&statsVar=Median_Age_Person%2C0%2C1__Count_Person%2C0%2C0"
+          );
+          // test if the statsVar info and statsVarTitle is updated
+          expect(wrapper.state().statsVarInfo).toEqual({
+            Median_Age_Person: { md: "", mprop: "age", pt: "Person", pvs: {} },
+            Count_Person: { md: "", mprop: "count", pt: "Person", pvs: {} },
+          });
+          expect(wrapper.state().statsVarTitle).toEqual({
+            Median_Age_Person: "Median age",
+            Count_Person: "Population",
+          });
+          expect(
+            pretty(wrapper.find("#chart-region").getDOMNode().innerHTML)
+          ).toMatchSnapshot();
```

Comment:
how does this test know which snapshot file to compare?

Reply:
The snapshot file has the file name: TestFileName.snap. If multiple snapshots are created within one test, the snapshots would have the name "test name + an index ", for example "`Single place and single stats var 1`"